docs/gateway-api/gateway-api-v1beta1.md

```diff
@@ -228,22 +232,32 @@ AKO accepts the following Gateway configuration for this release:
   2. Gateway MUST NOT contain protocols other than HTTP or HTTPS.
   3. Gateway MUST contain a hostname. Hostname as `*` is not supported and `*.domain` is supported.
   4. Gateway MUST NOT contain TLS modes other than `Terminate`.
+  5. If the Secret specified in Gateway-> listeners-> TLSConfig does not exist, our implementation will attach `SystemDefaultCert` to the Gateway. Once Secret is created, the `certRef` will be updated accordingly.
+  6. Gateway CAN have multiple listeners with same/overlapping hostname. 
+  7. Two Gateways MUST NOT have listeners with same/overlapping hostname.
+  8. `Gateway-> listeners-> allowedRoutes-> namespaces-> from` with value `selector` is not supported.
+  9. `Gateway-> listeners-> allowedRoutes-> namespaces-> selector` is not supported.
+ 10. Two Gateways with same name but in different namespace are supported.
```

Comment:
What is the use case of this? Will this complicate the things when reference grant feature is supported? This sounds like capability and not limitation.